mne/decoding/time_gen.py

```diff
@@ -1206,11 +1248,21 @@ class TimeDecoding(_GeneralizationAcrossTime):
                 each of the k-fold cross-validation classifiers, and average
                 these predictions into a single estimate per sample.
 
-        Default: 'cross-validation'
+        Defaults to 'cross-validation'.
     scorer : object | None | str
-        scikit-learn Scorer instance or str type indicating the
-        name of the scorer such as `accuracy`, `roc_auc`. If None,
-        set to accuracy_score.
+        scikit-learn Scorer instance or str type indicating the name of the
+        scorer such as `accuracy`, `roc_auc`. If None, set to accuracy_score.
+    score_mode : {'per-fold', 'mean-across-folds', 'no-cv'}
+        Determines how the scorer is estimated:
+
+            ``per-fold`` : returns the score obtained in each fold.
```

Comment:
you need to start these lines with `*` to get a bulleted list, this is what it looks like under this PR currently:

https://circle-artifacts.com/gh/mne-tools/mne-python/766/artifacts/0/home/ubuntu/mne-python/doc/_build/html/generated/mne.decoding.GeneralizationAcrossTime.html#mne.decoding.GeneralizationAcrossTime